docs/docs/intro.md

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Looks like we have duplicated docs folder inside another docs folder. Let's rename the top level docs to website